players/serializers.py

```diff
@@ -1,13 +1,54 @@
 from rest_framework import serializers
-from mobile_api.player_dao import PlayerDao
+# from mobile_api.player_dao import PlayerDao
+from players.models import Player
+from django.contrib.auth.models import User
+
+
+class PlayerCreateSerializer(serializers.Serializer):
+    def create(self, validated_data):
+        userauth = User.objects.create_user(username=self.data.get('player_id'),
+                                            password=self.data.get(
+                                                'password'), email=self.data.get('email'),
+                                            is_active=1, is_superuser=0)
+        player = Player(id=userauth, nickname=self.data.get("nickname"),
+                        phone_number=self.data.get("phone_number"),
+                        email=self.data.get("email"))
+        player.save()
+
+        return Player.objects.create(**validated_data)
 
```

Comment:
This serializer has not use, you forgot to delete it after implement RegisterSerializer